src/index.js

```diff
@@ -0,0 +1,9 @@
+import AdConnectorSourceId from './interface/AdConnectorSourceId'
+import AdLoadable from './interface/AdLoadable'
+import AdViewable from './interface/AdViewable'
+
+export {
+  AdConnectorSourceId,
```

Comment:
After talking about id source, this could be deleted

Reply:
Done